### DISCUSSION TITLE
How to couple aux variable of current timestep

### DISCUSSION THREAD

**Discussion by xiaojbing** [idaholab/moose#23536] [Q&A General]
By defalult，coupling aux variable is the old value of last timestep, right? So, How to couple aux variable of current timestep?

**Comment by GiudGiud** [comment]
Hello

No coupledValue is by default the current timestep, and for an implicit time integration scheme it's actually more the "next" time step.

To get the old value you use coupleValuesOld or coupledValuesOlder for the one before that

Guillaume
